src/plugins/editor/components/editor.jsx

```diff
@@ -162,6 +162,11 @@ export default function makeEditor({ editorPluginsToRun }) {
     }
 
     updateYaml = () => {
+      if(this.editor.getValue() === this.yaml[0]) {
```

Comment:
I believe this is addressed by the https://github.com/swagger-api/swagger-editor/blob/271a02dd9130a93a3fe7da851fc57c5195dae9b5/src/plugins/editor/components/editor.jsx#L229 change.
But simply because I don't know if it has, is why we're adding a bunch of tests based on this file.

Can be tracked here: https://github.com/swagger-api/swagger-editor/compare/master...feature/add-unit-test-to-editor